tasks-by-users/AbbasShaikh0304/06-CSS Grid- Masonary/style.css

```diff
@@ -0,0 +1,78 @@
+* {
+  margin: 0%;
+  padding: 0%;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+body {
+  display: flex;
+  justify-content: center;
+  align-self: center;
+
+  min-height: 100vh;
+
+  background: #222;
+}
+.container {
+  display: grid;
+
+  position: relative;
+  max-width: 100%;
+
+  margin: 40px;
+
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  grid-template-rows: minmax(300px, auto-fill);
+  grid-auto-flow: dense;
+  grid-gap: 10px;
+}
+.container .box {
+  display: grid;
+
+  padding: 20px;
+
+  font-size: 20px;
+  place-items: center;
+  text-align: center;
+
+  color: #fff;
+  background: #333;
+
+  transition: 0.5s;
+}
+.container .box:hover {
+  background: #e91e63;
+}
+.container .box img {
+  position: relative;
+
+  max-width: 100px;
+
+  margin-bottom: 10px;
+}
+.container .box:nth-child(1) {
+  grid-column: span 2;
+  grid-row: span 1;
+}
+.container .box:nth-child(2) {
```

Comment:
```suggestion
.container .box:nth-child(2),
.container .box:nth-child(4) {
```

Reply:
Thankyou for you suggestions. Commited the Changes @AlishaPrasad